client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss

```diff
@@ -187,6 +187,28 @@ $design-button-primary-color: rgb( 17, 122, 201 );
 		}
 	}
 
+	/**
+	 * Blank Canvas Button
+	 */
+	.build-your-own-cta-wrapper {
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+		align-items: center;
+		padding: 64px;
+		grid-column: 1 / span 3;
+
+		.build-your-own-cta_title {
+			margin-top: 32px;
+			margin-bottom: 8px;
+		}
+
+		.build-your-own-cta__button {
+			margin-top: 32px;
+			padding: 10px 80px;
```

Comment:
I think maybe setting the `min-width` might be better. If we set the horizontal padding and the text is longer after the translation, the width of the button would be longer